src/CarlaCDASimAdapter.py

```diff
@@ -79,13 +82,13 @@ def __connect(self):
         
     def __get_simulated_sensor(self, infrastructure_id, sensor_id):
         sensor = self.__api.get_simulated_sensor(infrastructure_id, sensor_id)
-        logging.debug(f"Retrieved sensor : {sensor.get_id()} for infrastructure {infrastructure_id}")
+        logging.info(f"Retrieved sensor : {sensor.get_id()} for infrastructure {infrastructure_id}")
         return str(sensor.get_id())
         
     def __get_detected_objects(self, infrastructure_id, sensor_id):
         detected_objects = self.__api.get_detected_objects(infrastructure_id, sensor_id)
         return_json = str(SimulatedSensorUtils.serialize_to_json(detected_objects))
-        logging.debug(f"Sensor {sensor_id} detected objects {return_json}")
+        logging.info(f"Sensor {sensor_id} detected objects {return_json}")
```

Comment:
Setting this to INFO level is going to be really noisy. Why are we changing it?

Reply:
I think for this service we want a log level that has this information and does not have the request information from the xmlrpc server. I can change it back

Reply:
Do we need XML-RPC logging? The two registered functions have their own logging statements, so it should indicate an RPC issue if those logs don't appear.

Reply:
One log statement is about connection while the other is about returned content. I think they are both valuable.

Reply:
That makes sense.

If the sensor detected objects information is useful only while debugging, it should be set to debug level. Otherwise, info level is fine. I'll approve it and leave it up to you to decide which is more appropriate.